rules/windows/privilege_escalation_printspooler_malicious_registry_modification.toml

```diff
@@ -10,7 +10,7 @@ Detects attempts to exploit privilege escalation vulnerabilities related to the
 information refer to CVE-2021-34527 and verify that the impacted system is investigated.
 """
 from = "now-9m"
-index = ["logs-endpoint.events.*", "logs-windows.*"]
+index = ["logs-endpoint.events.*"]
```

Comment:
The integration is only parsing DWORD and QWORD registry values, so registry.data.strings will always be null here. Dropping Support